app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+    @post = Post.find(params[:post_id])
+    @user = User.find(params[:user_id])
```

Comment:
- [ ] Why do you have this code on line 4 and 5, when you already have it on line 9 and 10.  I think it is unnecessary, so I suggest you remove line 4 and 5.